cpp/roaring64map.hh

```diff
@@ -31,8 +31,9 @@ namespace roaring {
 
 using roaring::Roaring;
 
-class Roaring64MapSetBitForwardIterator;
 class Roaring64MapSetBitBiDirectionalIterator;
+typedef Roaring64MapSetBitBiDirectionalIterator
```

Comment:
I think it would be useful to leave a comment about the purpose of this typedef, given that it's no longer used elsewhere in this file.

Reply:
Done